ts/test/test-init-config.ts

```diff
@@ -70,7 +79,15 @@ describe('initConfig', () => {
       zone: 'zone',
       projectId: 'fake-projectId'
     };
-    const initializedConfig = await initConfig(config);
+    const profiler: Profiler|undefined = await createProfiler(config);
+    if (profiler === undefined) {
+      assert.ok(true, 'profiler should be created successfully.');
+      return;
+    }
+    const initializedConfig = profiler.config;
+    // remove interceptors_ field which is added to profiler.config.
+    // tslint:disable-next-line: no-any
+    delete (initializedConfig as any).interceptors_;
```

Comment:
It might better to call `common.util.normalizeConfig` on `internalConfigParams` above and avoid this complexity here in each of the tests.

Reply:
Done.